src/Microsoft.AspNetCore.Mvc.Razor/RazorView.cs

```diff
@@ -155,7 +165,13 @@ public class RazorView : IView
         private Task RenderPageCoreAsync(IRazorPage page, ViewContext context)
         {
             page.ViewContext = context;
+
+            _diagnosticSource.BeforeRazorView(page, context);
```

Comment:
The methods should probably be named `BeforeRazorPage` \ `AfterRazorPage` since they're logging individual page execution.